### PR TITLE
remmina 1.4.30 (new formula)

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/remmina.rb [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+  url "https://gitlab.com/Remmina/Remmina/-/archive/v1.4.29/Remmina-v1.4.29.tar.bz2"
+  sha256 "94d11d00602d12ed033b1018e42171dfd1b1a42081530c229e4a3988c4f3a3d9"
+  license "GPL-2.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "freerdp"
+  depends_on "gtk+3"
+  depends_on "json-glib"
+  depends_on "libvncserver"
+  depends_on "openssl@1.1"
```
Comment (chenrui333) [inline Formula/remmina.rb:13]:
```suggestion
  depends_on "openssl@3"
```

File: Formula/remmina.rb [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  depends_on "vte3"
+
+  on_linux do
+    depends_on "webkitgtk"
+  end
+
+  # Patches for 1.4.29, merged by upstream, to be removed with 1.4.30.
+  # https://gitlab.com/Remmina/Remmina/-/merge_requests/2469
+  # https://gitlab.com/Remmina/Remmina/-/merge_requests/2471
+  patch :DATA
```
Comment (chenrui333) [inline Formula/remmina.rb:25]: when would be the next release?

File: Formula/remmina.rb [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  url "https://gitlab.com/Remmina/Remmina/-/archive/v1.4.29/Remmina-v1.4.29.tar.bz2"
+  sha256 "94d11d00602d12ed033b1018e42171dfd1b1a42081530c229e4a3988c4f3a3d9"
+  license "GPL-2.0-only"
+
+  depends_on "cmake" => :build
+  depends_on "freerdp"
+  depends_on "gtk+3"
+  depends_on "json-glib"
+  depends_on "libvncserver"
+  depends_on "openssl@3"
```
Comment (chenrui333) [inline Formula/remmina.rb:13]: you can find the dependency tree issue with `brew deps --tree remmina`

Looks like `freerdp` and `libvncserver` are using `openssl@3` while `unbound`, `libevent` and etc are using `openssl@1.1`

Comment (mvzlb) [reply Formula/remmina.rb:13]: How can this be handled? Add `remmina` to `audit_exceptions/versioned_dependencies_conflicts_allowlist.json`?

**Pull request closed without merging**